### PR TITLE
Feat/guide images

### LINKED ISSUE #4414: Nice categories images for Suite Guide
Currently we use only smiple categories boxes that are black & white. Let's put mages there.

[FIGMA](https://www.figma.com/file/fu1f9wihpfaT37QzvCkl4K/UI-Sync?node-id=1%3A33)

image

Export:
[ikones.zip](https://github.com/trezor/trezor-suite/files/8427518/ikones.zip)

### LINKED ISSUE #5206: images not showing in Suite Guide
Per [Suite Guide documentation](https://github.com/trezor/trezor-suite-guide#not-all-types-of-rich-content-are-supported), images are supported and should be shown in Suite Guide. However, images are currently not showing.

The following pages currently contain images:
https://github.com/trezor/trezor-suite-guide/blob/master/en/suite-basics/early-access-program.md
https://github.com/trezor/trezor-suite-guide/blob/master/en/suite-basics/message-signing-and-verification/sign-and-verify-in-
https://github.com/trezor/trezor-suite-guide/blob/master/en/suite-basics/message-signing-and-verification/electrum-compatible-signatures.md

**Info:**
 - Suite version: web 22.4.0 (554a4780f0ea58d30b32c7e5b39e3aed65044504)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36
 - OS: MacIntel
 - Screen: 2560x1440

**How to reproduce**
1. Open Suite Guide 
2. Go to Suite Basics > Early Access Program
3. Observe the lack of images

**Screenshots**
Suite Guide vs Gitbook repo
Screenshot 2022-03-31 at 18 02 24

### PR DESCRIPTION
**Pull request by tomasklim**
closes #5206, closes #4414

### CODE REVIEW

File: packages/utils/src/resolveStaticPath.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const resolveStaticPath = (path: string) =>
+    `${process.env.ASSET_PREFIX || ''}/static/${path.replace(/^\/+/, '')}`;
```
Comment (karliatto) [inline packages/utils/src/resolveStaticPath.ts]: I do not think it is a good idea to use a process env variable in utils package, why not adding it as function argument?

The idea of utils package is that is "independent" of the context where it is run, so I rather would suggest a more pure function approach.

**Review by karliatto — CHANGES_REQUESTED** [review]
Looks good to me and it works as expected in browser as electron app.

File: packages/utils/src/resolveStaticPath.ts (hunk 1 above)
Comment (tomasklim) [reply packages/utils/src/resolveStaticPath.ts]: 58ba96db11b38d38d821ac54060a9b1f0646ed8f

I agree. However, `resolveStaticPath` is used on 50 places around the monorepo, so I would add `process.env.ASSET_PREFIX` as a fallback for the prefix param.

Comment (tomasklim) [reply packages/utils/src/resolveStaticPath.ts]: 58ba96db11b38d38d821ac54060a9b1f0646ed8f I renamed it to `pathPrefix`

**Review by karliatto — APPROVED** [review]
:+1:

**Pull request merged**